static/bidder-params/bwx.json

```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "BoldwinX Adapter Params",
+    "description": "A schema which validates params accepted by the BoldwinX adapter",
+    "type": "object",
+    "properties": {
+      "env": {
+        "type": "string",
+        "description": "BoldwinX environment",
+        "minLength": 1
```

Comment:
Why is environment required? From the examples, it looks like is meant to suggest staging or prod. Do you really want to make your publishers enter "prod" for most of the time? How about making this optional with a good default?